types/svgicons2svgfont/svgicons2svgfont-tests.ts

```diff
@@ -0,0 +1,16 @@
+import SVGIcons2SVGFont from 'svgicons2svgfont';
```

Comment:
```suggestion
import SVGIcons2SVGFont = require('svgicons2svgfont');
```
CJS import (with TS touch):
https://github.com/nfroidure/svgicons2svgfont#in-your-scripts